hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsDfsClient.java

```diff
@@ -921,8 +1006,8 @@ public AbfsRestOperation deletePath(final String path, final boolean recursive,
      */
     final List<AbfsHttpHeader> requestHeaders = (isPaginatedDelete(recursive,
         isNamespaceEnabled) && xMsVersion.compareTo(
-        ApiVersion.AUG_03_2023) < 0)
-        ? createDefaultHeaders(ApiVersion.AUG_03_2023)
+        AbfsHttpConstants.ApiVersion.AUG_03_2023) < 0)
```

Comment:
I have imported AbfsConstatnt.ApiVersion
We don;t need to qualify it now.
Thoughts on this?